docs/getting-started.md

```diff
@@ -205,6 +216,32 @@ cog push
 
 The Docker image is now accessible to anyone or any system that has access to this Docker registry.
 
+> **Note**
+> When you build an image, Cog copies large files to a separate layer before copying source code. This reduces the time needed to rebuild after making changes to code. For best results, put any weights, checkpoints, or other model files in a separate subdirectory.
```

Comment:
I love this documentation, but I'm missing something: What is the effect of putting weights and other files in a subdirectory? I think that should be clarified here.

Reply:
Ah, I think I understand your feedback. If it's the case that Cog copies large files before code, then there shouldn't be a problem with them co-mingling with code. The reason why this matters is an implementation detail of how Docker `COPY` commands work — that is.

Rather than wade into that, I rewrote the callout to be explicit about the contract. "If you do this, then Cog can do this optimization".